docs/usage.md

```diff
@@ -905,15 +905,19 @@ Turn on the metagenomic screening module.
 
 #### `--metagenomic_tool`
 
-Specify which taxonomic classifier to use. The only option avaliable is currently 'malt'.
+Specify which taxonomic classifier to use. There are two options avaliable:
+- `malt` : more can be seen in the [MALT documentation](http://ab.inf.uni-tuebingen.de/data/software/malt/download/manual.pdf)
+:warning: **Important** It is very important to run `nextflow clean -f` on your nextflow run directory once completed. RMA6 files are VERY large and are _copied_ from a `work/` directory into the results folder. You should clean the work directory with the command to ensure non-redundency and large HDD footprints!
+- `kraken` with [Kraken2](https://ccb.jhu.edu/software/kraken2) 
 
-More can be seen in the [MALT documentation](http://ab.inf.uni-tuebingen.de/data/software/malt/download/manual.pdf)
+#### `--metagenomic_min_support_reads`
 
-:warning: **Important** It is very important to run `nextflow clean -f` on your nextflow run directory once completed. RMA6 files are VERY large and are _copied_ from a `work/` directory into the results folder. You should clean the work directory with the command to ensure non-redundency and large HDD footprints!
+Specify the minimum number of reads a given taxon is required to have to be retained as a positive 'hit'. 
+For malt, this only applies when `--malt_min_support_mode` is set to 'reads'. Default: 1 .
 
 #### `--database`
 
-Specify the path to the _directory_ containing your taxonomic classifer's database.
+Specify the path to the _directory_ containing your taxonomic classifer's database (malt or kraken).
```

Comment:
You need to be clearer here. For MALT I just ask for the directory, but it appears you also allow supplying a gzipped tar file? 

Maybe you could describe for MALT that this should be a directory _only_, whereas for Kraken it can also be in files in XYZ forms?

Reply:
Improved to make it clearer